compiler/test/stdlib/path.test.gr

```diff
@@ -0,0 +1,165 @@
+import Path from "path"
+import Result from "result"
+import Option from "option"
+import List from "list"
+
+// test that a variety of properties hold for valid relative/absolute paths
+let relTests = [
+  (
```

Comment:
I'm wondering if these should be a record like `TestDsl` or something? I keep mixing up the tuple meanings while reviewing

Reply:
Yeah good point, initially these tuples started out with only 3 elements but then grew into these monstrosities :smile: